dcos/http.py

```diff
@@ -182,16 +205,7 @@ def request(method,
     if 'headers' not in kwargs:
         kwargs['headers'] = {'Accept': 'application/json'}
 
-    if verify is None and constants.DCOS_SSL_VERIFY_ENV in os.environ:
-        verify = os.environ[constants.DCOS_SSL_VERIFY_ENV]
-        if verify.lower() == "true":
-            verify = True
-        elif verify.lower() == "false":
-            verify = False
-
-    # Silence 'Unverified HTTPS request' and 'SecurityWarning' for bad certs
-    if verify is not None:
-        silence_requests_warnings()
+    verify = _verify_ssl(verify)
```

Comment:
Sounds good. I still would like us to get rid of the side effect in `_verify_ssl` and maybe move it to `_request` what do you think?